lib/matplotlib/tests/test_axes.py

```diff
@@ -22,6 +22,10 @@
 import warnings
 from matplotlib.cbook import IgnoredKeywordWarning
 
+# Note: Some test cases are run twice: once normally and once with labeled data
```

Comment:
Unfortunately, I think we need to come up with a better test strategy. It doesn't scale well to double the runtime for what is effectively the same tests.

Can we not test something along the lines of:

```
def test_plot_datamodel():
    data = {'a': range(10)}
    with plt.plot('a', data=data) as a_thing:
        assert_equal(a_thing.signature, (data['a'],))